build.gradle

```diff
@@ -265,7 +265,7 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.8.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.8.4-Prerelease'
```

Comment:
This will need updating when the corresponding ccd-test-definitions updates are merged.

Reply:
7.8.5-pre in def store and 7.8.4-pre here are consistent?
Would be good to use a single latest stable release version in both.